.docs/main.rst

```diff
@@ -5,7 +5,7 @@
 
 .. image:: _images/flopylogo.png
 
-**Documentation for version 3.3.7 --- release candidate**
+**Documentation for version 3.5.0.dev0**
```

Comment:
Should this have a `.dev0`?  This is the only place I noticed it.  Everything else is just `3.5.0`.

Reply:
This gets updated when docs are built https://github.com/modflowpy/flopy/blob/338f6d073d6ba7edecc12405f3da83ea51e6cc26/.docs/conf.py#L33

however maybe in future it would be cleaner to sub it in with everything else via `scripts/update_version.py`